minfraud/webservice.py

```diff
@@ -251,10 +253,14 @@ def __init__(  # pylint: disable=too-many-arguments
           This sets both the connect timeout and the read timeout. The default is
```

Comment:
Pre-existing, but we say "seconts" in the above line (in GeoIP2-python too actually).